requirements.txt

```diff
@@ -1,3 +1,4 @@
+manifest-tool
```

Comment:
for next version this should be fixed to only accept major-version 1, if there is version 2, it should not automatically try to use it.

Reply:
It might be better to try to fix limit the versions of other dependencies at least a bit.